internal/pipeline/testdata/app-with-db/configs/db/Kptfile

```diff
@@ -0,0 +1,19 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: db
+pipeline:
```

Comment:
you want to test:

- a generator function in subpackage `db`
- a generator function in package `app`
- pipeline that uses `configs` to exercise filtering meta resources.
- validation functions: shouldn't be allowed to mutate resources
- failure cases

Reply:
This is not resolved yet. I will be addressing this soon and capture that in a separate commit.

Reply:
Added testcases for filtering meta resources test case and generator in the subpackage `db` . Rest of the test cases I will be adding in the separate PR.